integration-tests/standalone-asm/index.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const options = {
+  appsec: {
+    enabled: true
+  },
+  experimental: {
+    iast: {
+      enabled: true,
+      requestSampling: 100
+    }
+  }
+}
+
+if (process.env.AGENT_PORT) {
+  options.port = process.env.AGENT_PORT
+}
+
+if (process.env.AGENT_URL) {
+  options.url = process.env.AGENT_URL
+}
+
+const tracer = require('dd-trace')
+tracer.init(options)
+
+const http = require('http')
+const express = require('express')
+const app = express()
+
+const valueToHash = 'iast-showcase-demo'
+const crypto = require('crypto')
+
+app.get('/', (req, res) => {
+  res.status(200).send('hello world')
+})
+
+app.get('/login', (req, res) => {
+  tracer.appsec.trackUserLoginSuccessEvent({ id: req.query.user })
+  res.status(200).send('login')
+})
+
+app.get('/sdk', (req, res) => {
+  tracer.appsec.trackCustomEvent('custom-event')
+  res.status(200).send('sdk')
+})
+
+app.get('/vulnerableHash', (req, res) => {
+  const result = crypto.createHash('sha1').update(valueToHash).digest('hex')
+  res.status(200).send(result)
+})
+
+app.get('/propagation-with-event', async (req, res) => {
```

Comment:
I think it does...

Reply:
not support for it, it just ignores the result of the function, so the promise response is ignored. But if the only goal is use `async` to prevent using `.them()`..., I think that it is ok. (promise rejection is not going to be managed as Error throw, but for these tests... i think that it is ok)